### DISCUSSION TITLE
v2.16.2-beta

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#473] [General]
Fixes.

**Full Changelog**: https://github.com/Mygod/VPNHotspot/compare/v2.16.1...v2.16.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.2'>v2.16.2-beta</a>.</em>

**Comment by 006hjy** [comment]
I use KernelSU to get root permission
Failed to launch root daemon
can you fix it?

**Reply by Mygod** [reply]
What device are you using? Android 13? Does this only occur with KernelSU? Have you tried other SU?
